libminifi/include/utils/Id.h

```diff
@@ -65,10 +67,11 @@ class Identifier {
 
   bool operator!=(const Identifier& other) const;
   bool operator==(const Identifier& other) const;
+  bool operator<(const Identifier& other) const;
 
   bool isNil() const;
 
-  std::string to_string() const;
+  SmallString<36> to_string() const;
```

Comment:
This should get a comment explaining the reason for using `SmallString<36>` instead of `std::string` to avoid a future refactoring changing it back to `std::string`.

Reply:
added a comment